kahuna/public/js/services/api/collections-api.js

```diff
@@ -127,6 +127,35 @@ collectionsApi.factory('collections',
             });
     }
 
+    function findCollectionResource(image, collectionToMatch){
+        return image.data.collections.filter(collection => {
```

Comment:
You could use a [`find`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find)